nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/pom.xml

```diff
@@ -74,6 +74,10 @@
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-kerberos-user-service-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-kerberos-user-service-api</artifactId>
```

Comment:
Is there a reason for introducing this dependency declaration? It appears to to duplicate the dependency directly above.

Reply:
Thanks @exceptionfactory for spotting this, removed duplicate dependency.